src/components/Toast/Toast.module.css

```
@@ -25,11 +22,17 @@
 
   border-style: solid;
   border-width: var(--eds-border-width-sm);
-  border-left-width: var(--eds-border-width-lg); /* 3 */
+  /**
+  * The thick left border operates as a foreground band of color, 
+  * so foreground rather than border is used
+  */
```

Small annoying thing but the multi-line CSS comment blocks, everything after the first line is indented by 1 space.
```suggestion
  /**
   * The thick left border operates as a foreground band of color, 
   * so foreground rather than border is used
   */
```

---

Indentation fixed in next commit thanks!